core/src/test/java/com/linecorp/armeria/server/ServerTest.java

```diff
@@ -331,6 +332,34 @@ public void duplicatedPort() {
                 .hasCauseInstanceOf(IOException.class);
     }
 
+    @Test
+    public void testActiveLocalPort() throws Exception {
+        final int httpPort = 8080;
+        final int httpsPort = 8443;
+        final Server server = new ServerBuilder()
```

Comment:
This will make the test fail on a system that runs a server on this port. Could you use port number 0 and just check if activeLocalPort() returns a positive integer?

Reply:
@trustin I missed that :( changed again!